### DISCUSSION TITLE
making Windows installer on Linux?

### DISCUSSION THREAD

**Discussion by J-Kappes** [QuasarApp/CQtDeployer#698] [Q&A] [answered]
I used MXE to make an app.exe and successfully produced a deployed app.exe with
`cqtdeployer -bin app.exe -qmake /path/to/qmake`
Then, executing the same command with `qif` appended produced a .run installer. Is there a way to make an .exe installer instead?

**Comment by EndrII** [comment]
Yes, it is possible. You can call binarycreator.exe instead of the Linux version of the binarycreator tool.

But at current time CQtDeployer not support overriding of the command and I don't  have a free time to add new option. 
maybe you can add new option for cqtdeployer for overriding run command ?

If yes, I can create a new issue with dital description what you need to do

**Reply by J-Kappes** [reply]
I would be willing to devote 16 hours to that. However, I'm very inexperienced and can't promise any results 😬

**Comment by EndrII** [comment]
Good news, I found free time and will add new option itself
See #699

**Comment by EndrII** [comment, marked as answer]
Update cqtdeployer to [v1.5.4.16](https://github.com/QuasarApp/CQtDeployer/releases/tag/v1.5.4.16) and do this:

* install wine
* install windows binarycreator
* run cqtdeployer with the -binarycreator option

```bash
cqtdeployer -bin myBin -qmake myQmake ... -binarycreator 'wine path/to/win/binarycreator.exe' qif  
```


``` bash
~$ cqtdeployer help -binarycreator ''
Info: Print info ...
Information about using options
--------------------------------------------------------------------------------
  -binarycreator {binarycreator command}: Sets new binarycreator command.
                              : Example: cqtdeployer -bin my.exe qifw
                              : -binarycreator 'wine path/to/binarycreator.exe'
--------------------------------------------------------------------------------

```
